dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -40,7 +40,7 @@ excludeModule.desc=Module names, one per element, that are to be excluded from t
 openAPIVersion=OpenAPI specification version
 openAPIVersion.desc=The version of the OpenAPI specification that should be used for the OpenAPI document. mpOpenAPI-4.0 defaults to "3.1" and also supports "3.0". Earlier versions only support "3.0".
 
-info=OpenAPI document info
+info=OpenAPI Document Info
```

Comment:
Hi @Azquelt - the other  attributes in this file are all named in sentence case, which is consistent with IBM-Style

Reply:
The automated checking dinged me for it not being title case because it's the name of an element, rather than an attribute.

Usage would be something like this:
```xml
<mpOpenAPI>
  <info title="My API"
        description="Example description"
        version="1.0"/>
</mpOpenAPI>
```

Reply:
ah ok i see now- no change needed